blackbox/docs/admin/auth/hba.txt

```diff
@@ -160,3 +159,32 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
```

Comment:
also move the "initially" so it reads "authentication with the user .. will be attempted initially"